app/src/main/java/com/nervousfish/nervousfish/activities/ContactActivity.java

```diff
@@ -59,22 +65,32 @@ public void onClick(final SweetAlertDialog sDialog) {
                         .show();
             }
         });
+    }
 
-        //Retrieve the contacts from the database
-        getKeys();
-
-        lv.setAdapter(new ArrayAdapter<>(this,
-                android.R.layout.simple_list_item_1, keys));
-
+    /**
+     * Set the name of the {@link Contact} to the {@link ContactActivity}.
+     *
+     * @param contact The {@link Contact}.
+     */
+    private void setName(final Contact contact) {
```

Comment:
I don't find this really intuitive, I would just make the parameter a string, or do you think otherwise?